src/app/page.tsx

```diff
@@ -1,95 +1,30 @@
-import Image from "next/image";
-import styles from "./page.module.scss";
+"use client";
```

Comment:
Why do you need a page page? It does an empty request then after typing in SearchPanel it makes a second request and push route to a search page, doesn’t it? Please rewrite it with one page.